tests/roles/swift_migration/tasks/main.yaml

```diff
@@ -25,7 +31,9 @@
     {{ oc_header }}
     oc debug --keep-labels=true job/swift-ring-rebalance -- /bin/sh -c '
     swift-ring-tool get
-    swift-ring-tool drain 172.20.0.100
+    for node in {{ swift_node_list_result.stdout }}; do
```

Comment:
The content of swift_node_list_result.stdout is a newlines separate list, which breaks the shell interpretation as the items behind the first one creates  new lines, apparently. Or at least it looks like from the logs:

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "\noc debug --keep-labels=true job/swift-ring-rebalance -- /bin/sh -c '\nswift-ring-tool get\nfor node in 172.20.0.103\n172.20.0.104\n172.20.0.105; do\n  swift-ring-tool drain ${node}\ndone\nswift-ring-tool forced_rebalance\nswift-ring-tool push'\n", "delta": "0:00:02.250242", "end": "2024-11-08 08:26:10.942126", "msg": "non-zero return code", "rc": 1, "start": "2024-11-08 08:26:08.691884", "stderr": "Starting pod/swift-ring-rebalance-debug-kwp8d, command was: /usr/local/bin/swift-ring-tool all\n\nRemoving debug pod ...\nerror: non-zero exit code from debug container", "stderr_lines": ["Starting pod/swift-ring-rebalance-debug-kwp8d, command was: /usr/local/bin/swift-ring-tool all", "", "Removing debug pod ...", "error: non-zero exit code from debug container"], "stdout": "Total bytes read: 122880 (120KiB, 33MiB/s)\n/bin/sh: -c: line 4: syntax error near unexpected token `172.20.0.104'\n/bin/sh: -c: line 4: `172.20.0.104'", "stdout_lines": ["Total bytes read: 122880 (120KiB, 33MiB/s)", "/bin/sh: -c: line 4: syntax error near unexpected token `172.20.0.104'", "/bin/sh: -c: line 4: `172.20.0.104'"]}